umm-spec-lib/src/cmr/umm_spec/migration/version/service.clj

```diff
@@ -173,6 +174,198 @@
     "WEB SERVICES"
     service-type))
 
+(defn create-main-url-for-1_3
+  "When migrating from UMM-S v1.2 to UMM-S v1.3 verison 1.2 RelatedURLs needs to be
+   converted to the new URL element.  Take the first DistributionURL element from
+   RelatedURLs and convert it to the new URL type."
+  [s]
+  (let [url (->> (:RelatedURLs s)
+                 (filter #(= (:URLContentType %) "DistributionURL"))
```

Comment:
You can replace this filter-first with 
```clojure
(some #(= (:URLContentType %) "DistributionURL"))
```

Reply:
the some function is just returning the boolean true and not a map within a vector

Reply:
(->> (:RelatedURLs s1)
     (some #(= (:URLContentType %) "DistributionURL")))
cmr.umm-spec.migration.version.service=>
true
--------------------
Where (:RelatedURLs s1) is
(:RelatedURLs s1)
cmr.umm-spec.migration.version.service=>
[{:Subtype "USER'S GUIDE",
  :URLContentType "PublicationURL",
  :Description "User Guide",
  :Type "VIEW RELATED INFORMATION",
  :URL "http://docserver.gesdisc.eosdis.nasa.gov/repository/Mission/AIRS/3.3_ScienceDataProductDocumentation/3.3.4_ProductGenerationAlgorithms/V6_L3_User_Guide.pdf"}
 {:Subtype "OPENDAP DATA",
  :URLContentType "DistributionURL",
  :Description "OPeNDAP Service for AIRX3STD.006",
  :Type "GET SERVICE",
  :URL "https://acdisc.gesdisc.eosdis.nasa.gov/opendap/Aqua_AIRS_Level3/AIRX3STD.006/"}
 {:Subtype "USER'S GUIDE",
  :URLContentType "PublicationURL",
  :Description "User Guide",
  :Type "VIEW RELATED INFORMATION",
  :URL "http://docserver.gesdisc.eosdis.nasa.gov/repository/Mission/AIRS/3.3_ScienceDataProductDocumentation/3.3.4_ProductGenerationAlgorithms/V6_L3_User_Guide.pdf"}]